src/Analyzers/Core/CodeFixes/PopulateSwitch/AbstractPopulateSwitchCodeFixProvider.cs

```diff
@@ -222,7 +222,7 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         private class MyCodeAction : CustomCodeActions.DocumentChangeAction
         {
             public MyCodeAction(string title, Func<CancellationToken, Task<Document>> createChangedDocument)
-                : base(title, createChangedDocument)
+                : base(title, createChangedDocument, title)
```

Comment:
Can you check the FixAll behavior before and after this fix? I am wondering how it was working if same equivalence key was used for multiple registered code actions for same diagnostic.

Reply:
We have tests for this in PopulateSwitchStatementTests_FixAllTests.cs.  I imagine it wasn't working properly either.  Possibly another issue with the testing harness?